modules/troubleshooting-openshift-install-command-issues.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting-installations.adoc
+
+[id="troubleshooting-openshift-install-command-issues_{context}"]
+= Troubleshooting 'openshift-install' command issues
+
+If you experience issues running the `openshift-install` command, check the following:
+
+* The installation has been initiated within 24 hours of ignition configuration file creation (`./openshift-install create ignition-configs --dir=./install_dir`).
```

Comment:
Consider putting that full command in a separate code block, and mentioning that it's the command used to create the ignition config files. The user might try it, thinking that it's a command to see when they were created.

Reply:
Thank you, that's a great suggestion. Resolved.